.vscode/launch.json

```diff
@@ -51,7 +50,7 @@
                 "${file}"
             ],
             "env": {
-                "BICEP_TRACING_ENABLED": "true"
+                "BICEP_TRACING_ENABLED": "true",

```

Comment:
nit: Technically this isn't allowed in JSON, althgouh vscode treats launch.json as JSONC, which I guess allows it as well as comments.  Change or leave as is, I don't care.

Reply:
I will take it into consideration on next PRs, I rather not have to run a whole build just for handling this detail

Reply:
More of an FYI.  I said I don't care, so feel free to just resolve without commenting.